_posts/en/pages/2016-01-01-index.md

```diff
@@ -5,9 +5,44 @@ lang: en
 title: Bitcoin
 name: index
 permalink: /
-version: 2
+version: 3
 tags: [bitcoin, bitcoin core]
-translated: false
+
+## Strings to localize on translated versions of this page
+bitcoin-core: "Bitcoin Core"
+download-bitcoin-core: "Download Bitcoin Core"
+about1: >
+  The Bitcoin Core project provides software that helps you use Bitcoin.
+  Its flagship program is a full node that finds the valid chain
+  of blocks with the most Proof-of-Work (PoW).  The users of Bitcoin
+  Core only accept transactions on that chain, making it the Bitcoin
+  block chain that everyone else wants to use.
+
+about2: >
+  It is these users who keep Bitcoin decentralized. They
+  individually run their own Bitcoin Core full nodes, and each of
+  those full nodes separately follows the exact same rules to decide
```

Comment:
```suggestion
  those full nodes separately follow the exact same rules to decide
```

Reply:
I believe the agreement here is correct.

Reply:
Full nodes follow
Full node follows.

"follow" should be singular

Reply:
It still sounds wrong to me in the singular, but I don't think it's important, so I'll change it.